degirum_tools/line_count.py

```diff
@@ -54,25 +54,50 @@ class LineCounter(ResultAnalyzerBase):
 
     def __init__(
         self,
-        lines: List[tuple],
+        lines: np.ndarray,
         anchor_point: AnchorPoint = AnchorPoint.BOTTOM_CENTER,
+        whole_trail: bool = True,
+        count_first_crossing: bool = True,
+        absolute_directions: bool = True,
+        accumulate: bool = True,
         *,
         per_class_display: bool = False,
+        show_overlay: bool = True,
+        window_name: Optional[str] = None,
     ):
         """Constructor
 
         Args:
             lines (list[tuple]): list of line coordinates;
-                each list element is 4-element tuple of (x1,y1,x2,y2) line coordinates
+                each list element is 2-element tuple of tuples, each of which is an (x,y) line coordinate

```

Comment:
Wow! You changed public API incompatible way! Better not to do that!

Reply:
I chose this format because was convenient for calculating vectors. I changed it back though now.